src/overlays/actors/ovl_Demo_Im/z_demo_im.c

```diff
@@ -166,44 +166,44 @@ void DemoIm_UpdateCollider(DemoIm* this, PlayState* play) {
 
 void func_80984DB8(DemoIm* this) {
     s32 pad[2];
-    Vec3s* vec1 = &this->unk_2D4.unk_08;
-    Vec3s* vec2 = &this->unk_2D4.unk_0E;
+    Vec3s* rotHead = &this->interactInfo.rotHead;
+    Vec3s* rotTorso = &this->interactInfo.rotTorso;
 
-    Math_SmoothStepToS(&vec1->x, 0, 20, 6200, 100);
-    Math_SmoothStepToS(&vec1->y, 0, 20, 6200, 100);
+    Math_SmoothStepToS(&rotHead->x, 0, 20, 6200, 100);
+    Math_SmoothStepToS(&rotHead->y, 0, 20, 6200, 100);
 
-    Math_SmoothStepToS(&vec2->x, 0, 20, 6200, 100);
-    Math_SmoothStepToS(&vec2->y, 0, 20, 6200, 100);
+    Math_SmoothStepToS(&rotTorso->x, 0, 20, 6200, 100);
+    Math_SmoothStepToS(&rotTorso->y, 0, 20, 6200, 100);
 }
 
 void func_80984E58(DemoIm* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
     s16 yawDiff;
-    s16 phi_a3;
+    s16 playerTrackOpt;
```

Comment:
`playerTrackOpt` vs `playerTrackingOpt`

I don't have an opinion but, consistency!

Reply:
at first glance i feel like in general this can be "mode" or "type" or something, but havent done a full read through of the system to give a confident suggestion

Reply:
Mode is definitely better than opt